### ISSUE TITLE
support, URL, FHIRBinaries, includedB64 for itemMedia

### ISSUE THREAD

**Issue by delcroip** [SwissTPH/pyfhirsdc#36]
```
    #TODO: support, URL, FHIRBinaries, includedB64
```

https://github.com/SwissTPH/pyfhirsdc/blob/af0491dd6da230aafc76d9e38d268187618683ec/pyfhirsdc/converters/extensionsConverter.py#L26

**Comment by rukshn** [comment]
Hello Patrick, I was studying the program during the weekend, and this is a really valuable tool. 

I was looking at this issue, is this to support base64 encoded media data and serve it via the `data` element of the `attachment` element of the `itemMedia` extension?

**Comment by delcroip** [comment]
Yes, i already did it but in an ad hoc tool because i don't want to bundle the images with the rest to avoid too big push

**Comment by rukshn** [comment]
Yes I agree. Even sending one image through REST will significantly increase the bundle size.

**Comment by delcroip** [comment]
Hello @rukshn 

the next task/dev that I might ask you to do  will be to add the support of:
- Measures (special kind of library, worksheet prefix "m")
- Test case - Reference patient :we need a way to define quickly patient basic data + past conditons 
- Test case - Refence Encounter: we need to define encounter + Observation (ideally using the WHO pseudo language "dataelementLabel" = true etc)
- migrate global IG to https://github.com/WorldHealthOrganization/smart-ig-starter-kit
- a way to inject  Use case in the IG (I will give you the file)

could you have a look and tell me with which one you want to start ?

br

**Comment by rukshn** [comment]
Hello @delcroip 

Can you share details in migrating global IG to the WHO smart IG starter kit. 

I will meanwhile look at the starter kit repository

**Comment by delcroip** [comment]
I have created an issue in the right repo

https://github.com/WorldHealthOrganization/smart-emcare/issues/212
